web/styles/popovers.css

```diff
@@ -1098,20 +1126,36 @@ ul {
     }
 }
 
-.personal-menu-actions {
+.navbar-dropdown-menu {
     display: flex;
     flex-flow: column nowrap;
+    margin: 0;
+    padding: 0;
+    border: solid 1px var(--color-border-dropdown-menu);
+    background-color: var(--color-background-dropdown-menu);
+    max-height: 85vh;
+    min-width: 230px;
+    overflow-x: hidden;
+    user-select: none;
+    border-radius: 6px;
+    box-shadow: var(--box-shadow-navbar-dropdown-menu);
 
-    .list {
+    .simplebar-content {
+        min-width: max-content;
```

Comment:
Let's note here that this is necessary to keep long menu items on a single line; I was about to say to strike it until I poked a little harder.

Reply:
Done. Thanks!